packages/primitives-core/src/css.js

```diff
@@ -121,5 +121,25 @@ function convertStyles(str: string) {
 
   parsedString.forEach(convertPropertyValue, stylePairs)
 
-  return transform(stylePairs)
+  try {
```

Comment:
aint sure entirely about this - but `primitives` might be used on web too, maybe this try/catch should be wrapped with development check? wdyt @mitchellhamilton ?

Reply:
Yeah, that might be a good idea.

Reply:
Gonna merge this regardless of this. I suspect that we might want to drop primitives & native packages in emotion@11 because it seems we don't have capacity to handle their issues.

The code as-is improves the story around those packages right now, so it IMHO deserves being merged.